README.md

```diff
@@ -82,7 +82,7 @@ api.track_events(event)
 api.track_events(event, allow_offline: true)
 
 # You can also track events and user attributes at the same time
-api.track_multi(event, attribute_hash)
+api.track_multi(events: event, user_attributes: attribute_hash)
```

Comment:
you need to update the documentation for the new `device_attributes` key, and you also need to update the line below about how to use the `force_anomalous_override` option.

Reply:
will do